src/components/base/OakMaxWidth/OakMaxWidth.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render } from "@testing-library/react";
+import { create } from "react-test-renderer";
+
+import { OakMaxWidth } from "./OakMaxWidth";
+
+describe("OakMaxWidth", () => {
+  it("renders", () => {
+    const { getByTestId } = render(<OakMaxWidth data-testid="test" />);
+    expect(getByTestId("test")).toBeInTheDocument();
+  });
+
+  it("matches snapshot", () => {
+    const tree = create(<OakMaxWidth />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it("has has max-width when screen width is large", () => {
```

Comment:
this assumes some implicit behaviour regarding media points in jest. Lets feed through a a value and make it more explicit.